spark/src/main/java/org/apache/iceberg/spark/actions/BaseRemoveOrphanFilesSparkAction.java

```diff
@@ -140,10 +142,20 @@ public BaseRemoveOrphanFilesSparkAction deleteWith(Consumer<String> newDeleteFun
 
   @Override
   public RemoveOrphanFiles.Result execute() {
-    JobGroupInfo info = newJobGroupInfo("REMOVE-ORPHAN-FILES", "REMOVE-ORPHAN-FILES");
+    JobGroupInfo info = newJobGroupInfo("REMOVE-ORPHAN-FILES",
+        String.format("Removing orphan files(%s) from %s", getDescription(), table.name()));
```

Comment:
Shouldn't this just be "getDescription()"?

Reply:
Fixed